FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2ListItemActivity.kt

```diff
@@ -253,18 +261,20 @@ private fun OneLineSimpleList() {
     return Column {
         ListItem.Item(
             text = primaryText,
+            onClick = {},
             leadingAccessoryContent = { GetAvatar(size = Size24, drawable.avatar_amanda_brady) },
             border = BorderType.Bottom,
             borderInset = XXLarge
         )
         ListItem.Item(
             text = primaryText,
+            onClick = {},
             leadingAccessoryContent = { GetAvatar(size = Size24, drawable.avatar_allan_munger) },
             border = BorderType.Bottom,
             borderInset = XXLarge
         )
         ListItem.Item(
-            text = primaryText,
+            text = primaryText+" (unclickable)",
```

Comment:
create a var for " (unclickable)" text and use it a multiple places.

Reply:
Done.